allennlp/nn/decoding/decoder_trainers/expected_risk_minimization.py

```diff
@@ -104,9 +112,12 @@ def _get_costs_by_batch(states: List[StateType],
             batch_costs[batch_index].append(cost)
         return batch_costs
 
-    def _get_best_action_sequences(self, finished_states: List[StateType]) -> Dict[int, List[int]]:
+    def _get_best_action_sequences(self,
+                                   finished_states: List[StateType],
+                                   max_num_decoded_sequences: int = 1) -> Dict[int, List[List[int]]]:
```

Comment:
Nit: This is a private method, and `max_num_decoded_sequences` is a member variable.  You don't need to pass it as an argument to this method.  Do you gain anything by passing it instead of just reading it from `self`?

Reply:
Yes, you're right. Fixed it.